qiskit/compiler/transpile.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2019, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""Circuit transpile function """
+import logging
+
+from qiskit import transpiler
+from qiskit.mapper import Layout
+
+
+logger = logging.getLogger(__name__)
+
+
+def transpile(circuits, transpile_config=None):
+    """Compile a list of circuits into a list of optimized circuits.
+
+    Args:
+        circuits (QuantumCircuit or list[QuantumCircuit]): circuits to compile
+        transpile_config (TranspileConfig): configuration for the transpiler
+
+    Returns:
+        circuits: the optimized circuits
+
+    """
+
+    # ------------
+    # This is a HACK while we are still using the old transpiler.
+    try:
+        initial_layout = transpile_config.initial_layout
+    except AttributeError:
+        initial_layout = None
+    try:
+        basis_gates = transpile_config.basis_gates
+    except AttributeError:
+        basis_gates = None
+    try:
+        coupling_map = transpile_config.coupling_map
+    except AttributeError:
+        coupling_map = None
+    try:
+        seed_mapper = transpile_config.seed_mapper
+    except AttributeError:
+        seed_mapper = None
```

Comment:
these lines can be more compact written like this:
```python
getattr(transpile_config, 'seed_mapper', None)
```

Reply:
I agree